src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -560,7 +561,7 @@ private static void WriteInitializer(INamedTypeSymbol typeSymbol, IndentedString
 					private readonly static IEventProvider _binderTrace = Tracing.Get(DependencyObjectStore.TraceProvider.Id);
 					private BinderReferenceHolder _refHolder;
 
-					public event Windows.Foundation.TypedEventHandler<DependencyObject, DataContextChangedEventArgs> DataContextChanged;
+					public event Windows.Foundation.TypedEventHandler<FrameworkElement, DataContextChangedEventArgs> DataContextChanged;
```

Comment:
@ebariche This change affects LOTS (almost all?) of the controls. How can I handle that in PackageDiffIgnore?